ball_box_canvas.js

```diff
@@ -21,6 +21,7 @@ let boxes = {};  // {1: [x, y, status <0: not opened, 1: opened>]}
 let gameRunning = false;
 let firstGameStarted = false;
 let drawingInterval
+let balls_collected = 0;
```

Comment:
Unnecessary. Check function updateGameStatus() and variable lastBoxOpen. If it goes to 23, you have the same result.